scripts/lint.py

```diff
@@ -342,6 +342,38 @@ def check_rule(self, ctx, rule):
         return False
 
 
+class FormatStringIncorrect(Lint):
```

Comment:
enforce three things:

1. strings should not start/end with `'`
2. strings with special characters must start/end with `"`
3. special characters must be escaped